stable/license/README.md

```diff
@@ -0,0 +1,126 @@
+# Install NuoDB License on Kubernetes
+
+When using the Helm package manager to deploy NuoDB, a license is not installed by default.
+
+To view the details such as date of expiry (`expires`), name of the license holder (`holder`), and the type of license (`type`) , run:
+
+```
+  nuocmd get effective-license
+```
+
+For example, if the *Enterprise License* is installed:
+
+```
```

Comment:
If you replace ` ``` ` with ` ```console `, then this will distinguish input from output. You can then write your example as follows:

```
$ nuocmd --show-json get effective-license 
{
  "decodedLicense": {
    "expires": "2025-01-31T15:33:40.465134",
    "holder": "3DS Solutions",
    "type": "ENTERPRISE"
  },
  "effectiveForDomain": true,
  "encodedLicense":
    "-----BEGIN LICENSE-----
    <base64-encoded data>
    -----END LICENSE-----"
}
```

Note that there is a dollar sign `$`.

This is how it would look (note the highlighting):

```console
$ nuocmd --show-json get effective-license
{
  "decodedLicense": {
    "expires": "2025-01-31T15:33:40.465134",
    "holder": "3DS Solutions",
    "type": "ENTERPRISE"
  },
  "effectiveForDomain": true,
  "encodedLicense":
    "-----BEGIN LICENSE-----
    <base64-encoded data>
    -----END LICENSE-----"
}
```